src/translate.js

```diff
@@ -1,14 +1,14 @@
 import curry from 'lodash.curry';
 import { connect } from 'react-redux';
 import { getP } from './selectors';
-import { isFunction, isString } from './private/utils';
+import { isFunction, isString, isObject } from './private/utils';
 
 const getDisplayName = Component => (
     Component.displayName || Component.name || 'Component'
 );
 
-const mapPolyglotToProps = polyglotScope => state => ({
```

Comment:
OK, I see what you mean. 
I was confused between `mapPolyglotToProps`, which is fine, and `mapPolyglotScopeToProps`, (because we changed `polyglotScope` argument to `options`), which would have been wrong. 
So, I guess `mapPolyglotToProps` is OK for me.